build_tools/github_actions/lint_buildifier.sh

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 The StableHLO Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This runs buildifier on relevant files
+# This runs markdownlint-cli with the specified files, using Docker.
+# If passing the files as a glob, be sure to wrap in quotes. For more info,
+# see https://github.com/igorshubovych/markdownlint-cli#globbing
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly STABLEHLO_ROOT_DIR="${SCRIPT_DIR}/../.."
+
+cd "$STABLEHLO_ROOT_DIR"
+
+targets=(
```

Comment:
maybe something that works forward for new files: 
https://stackoverflow.com/questions/4210042/how-do-i-exclude-a-directory-when-using-find